### PR TITLE
Topics: Merged topics #10

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

Merges 18 pickup topics A-M.

| old topic                                   | new topic                         |
| ------------------------------------------- | --------------------------------- |
| agile                                       | product-and-project-management    |
| agile-lean-community-of-practice            | product-and-project-management    |
| apps-challenges                             | challenges-and-prize-competitions   |
| architecture                                | information-architecture          |
| browsers                                    | software-engineering              |
| challenge-gov                               | challenges-and-prize-competitions |
| challenges                                  | challenges-and-prize-competitions |
| challenges-and-prizes-community-of-practice | challenges-and-prize-competitions |
| contact-center                             | contact-centers                    |
| government-contact-center-council           | contact-centers                    |
| government-contact-centers                  | contact-centers                    |
| pra                                         | information-collection            |
| kanban                                      | product-and-project-management    |
| lean                                        | product-and-project-management    |
| metadata                                    | search-engine-optimization        |
| plain-writing-act-of-2010                   | plain-language                    |
| prizes-and-competitions                     | challenges-and-prize-competitions   |

> [!NOTE]
> The following files contained updated `ref` shortcodes

- content/news/2016/09/2016-09-30-creating-wall-sized-interaction-at-the-smithsonians-national-air-and-space-museum.md
- content/news/2016/11/2016-11-28-gsa-presents-grace-hopper-day-hackathon.md 

### What To Test

- [ ] replace `old topic` with `new topic` in front matter
- [ ] delete `old topic` folder in `content/topics`
- [ ] update `ref` from old topic to new topic path

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

**Review by mejiaj — COMMENTED** [review]
In the table I see `contact-centers → contact-center`, but there are 38 references to the old topic when searching via: `^\s*-\s* (agile|agile-lean-community-of-practice|apps-challenges|architecture|browsers|challenge-gov|challenges|challenges-and-prizes-community-of-practice |contact-centers|government-contact-center-council|government-contact-centers|pra|kanban|lean|metadata|plain-writing-act-of-2010|prizes-and-competitions)$`

Is this entry accurate?

**Comment by nick-mon1** [comment]
`contact-centers` is the correct topic, this was an error in the original spreadsheet that had to be corrected and wasn't updated in the above table. Also corrected `challenge-and-prize-competition` to be `challenges-and-prize-competitions`. Table is now up-to-date.

**Comment by mejiaj** [comment]
@nick-mon1 thanks for that note. Can you look at the remaining conflicts?

**Comment by nick-mon1** [comment]
@mejiaj Merged conflicts resolved, ready for another review

**Review by clmedders — APPROVED** [review]
looks good

i tested to see that old topcis are not present with 
`^\s*-\s* (agile|agile-lean-community-of-practice|apps-challenges|architecture|browsers|challenge-gov|challenges|challenges-and-prizes-community-of-practice |government-contact-center-council|government-contact-centers|pra|kanban|lean|metadata|plain-writing-act-of-2010|prizes-and-competitions)$`

and also did clean up of repeated topcis

File: content/events/2019/11/2019-11-08-basic-scrum-i.md
Comment (ToniBonittoGSA) [inline content/events/2019/11/2019-11-08-basic-scrum-i.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/events/2019/11/2019-11-08-basic-scrum-ii.md
Comment (ToniBonittoGSA) [inline content/events/2019/11/2019-11-08-basic-scrum-ii.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/events/2019/11/2019-11-08-intro-kanban-i.md
Comment (ToniBonittoGSA) [inline content/events/2019/11/2019-11-08-intro-kanban-i.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/events/2019/11/2019-11-08-intro-kanban-ii.md
Comment (ToniBonittoGSA) [inline content/events/2019/11/2019-11-08-intro-kanban-ii.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/news/2020/04/2020-04-24-experimenting-with-micropurchase-threshold.md
Comment (ToniBonittoGSA) [inline content/news/2020/04/2020-04-24-experimenting-with-micropurchase-threshold.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/news/2020/10/2020-10-24-discovery-bpa-evaluation-process-how-to-operationalize-this-practice.md
Comment (ToniBonittoGSA) [inline content/news/2020/10/2020-10-24-discovery-bpa-evaluation-process-how-to-operationalize-this-practice.md:1]: Remove `project-management`; merged into `product-and-project-management`.

File: content/news/2023/12/2023-12-20-managing-a-high-performing-dream-team-in-the-federal-government.md
Comment (ToniBonittoGSA) [inline content/news/2023/12/2023-12-20-managing-a-high-performing-dream-team-in-the-federal-government.md:1]: Change `project-management` to `product-and-project-management`.

File: content/resources/18f-de-risking-guides.md
Comment (ToniBonittoGSA) [inline content/resources/18f-de-risking-guides.md:1]: Change `project-management` to `product-and-project-management`.

**Pull request merged**